src/components/DownloadAsCsv/DownloadAsCsv.tsx

```diff
@@ -3,7 +3,7 @@ import { useCallback } from "react";
 
 import { downloadAsCsv } from "lib/csv";
 
-import DownloadIcon from "img/ic_download.svg";
+import downloadIcon from "img/ic_download.svg";
```

Comment:
don't we need `?react` here?

Reply:
That's an intended fix: downloadIcon is used as a url, not as a component in the further code